### ISSUE TITLE
Unable to run under windows XP

### ISSUE THREAD

**Issue by movemove** [lxn/walk#156]
I've compiled a binary under windows 7 AMD64 machine with win32 golang environment, the executable would run under windows 7, but when I copy it to Windows XP (win32), it will show a black dos window and says "panic:".    Please advise. thanks

**Comment by lxn** [comment]
Please update your copies of the `win` and `walk` packages and try again.

**Comment by movemove** [comment]
lxn, thanks for the speedy response, I will try it as soon as I get my hands on a computer, and will update the issue promptly.

thanks again

**Comment by matadin** [comment]
I also had similar problems on WinXP (win32) several days ago, but now it is OK, several examples I've tried are running fine.

From my earlier error messages it seems that:
`MustGetProcAddress(libkernel32, "GetThreadUILanguage")`
and after I switched it off:
`MustGetProcAddress(libuser32, "AddClipboardFormatListener")`
panicked, now they are changed in `syscall.GetProcAddress(...)` functions.

According to [MSDN documentation](https://msdn.microsoft.com/en-us/library/windows/desktop/dd318129%28v=vs.85%29.aspx) those functions are not in kernel32.dll and user32.dll in WindowsXP, only in Vista and later. But now they don't panic.

I am happy you are supporting WindowsXP,
thanks
